flowzone.yml

```
@@ -1469,7 +1469,12 @@ jobs:
   website_build:
     name: Build website using Docusaurus
     runs-on: ${{fromJSON(inputs.runs_on)}}
-    needs: [project_types, versioned_source]
+    env:
+     CF_BRANCH: ${{ github.event.pull_request.head.ref }}
```

```suggestion
     CF_BRANCH: ${{ github.event.pull_request.head.ref || github.event.repository.default_branch }}
```
I think just having the default branch as a fallback here will handle tag events. We don't use this workflow internally but tags would be treated like a push to master.

---

@klutchell done! could you approve the PR please?

---

@ab77 has changes requested, I can override his review but let's give him a chance to clear it himself. LGTM though!